scripts/ci.release.js

```diff
@@ -79,8 +79,18 @@ function versionTagAndPublish() {
 	log(`Great success, much amaze`);
 }
 
+function releaseNpmTag() {
+	if (process.env.GIT_BRANCH === 'master') {
+		return 'latest';
+	} else if (process.env.RELEASE_NPM_TAG) {
+		return process.env.RELEASE_NPM_TAG;
```

Comment:
So this means we cannot release a tag from master... Do we not want to keep this option as an escape hatch? meaning, prioritize any value set for `RELEASE_NPM_TAG` over anything else (including the current branch)?